app/demo-rv64-qemu-supervisor/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "demo-rv64-qemu-supervisor"
```

Comment:
Is the `virt` dropped intentionally? We might have more than one rv64 qemu targets ;).

Reply:
Yes - but only to save my typing/log lines. I'll add it back in.

Reply:
We can probably rename to something shorter: rv64-virt and rv64-virt-s.